### PR TITLE
Fix use of GalaxyClient with username/password on py3

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: bioblend/galaxyclient.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def make_put_request(self, url, payload=None, params=None):
     @property
     def key(self):
         if not self._key and self.email is not None and self.password is not None:
-            unencoded_credentials = "%s:%s" % (self.email, self.password)
+            unencoded_credentials = b"%s:%s" % (self.email, self.password)
```
Comment (nsoranzo) [inline bioblend/galaxyclient.py]: This raises an exception for me:
`TypeError: unsupported operand type(s) for %: 'bytes' and 'tuple'`

what about modifying the next line instead to:
```python
authorization = base64.b64encode(unencoded_credentials.encode())
```

Comment (mvdbeek) [reply bioblend/galaxyclient.py]: I could have sworn I tested this under py3 as well :).

**Comment by mvdbeek** [comment]
I've rebased the commit with the right fix.

**Comment by nsoranzo** [comment]
Thanks @mvdbeek!

**Pull request merged**